src/util/constants.ts

```diff
@@ -56,3 +56,15 @@ export const BEHAVIOR_TYPES = [
 ];
 
 export const DISPLAY = ":99";
+
+export enum ExitCodes {
+  Success = 0,
+  GenericError = 1,
```

Comment:
Looks like we only use this if the crawl is terminated due to a signal, so maybe we can rename to something a little more specific?

```suggestion
  Terminated = 1,
```